source/_docs/metrics.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Metrics in the Site Dashboard
+description: Measure your site's traffic with the Metrics tool, found in the Live environment of the Site Dashboard.
+tags: [performance, logs]
```

Comment:
@ari-gold happy to talk through the tags if you'd like to weigh in. Tags are used to populate doc lists on "topic index pages" such as https://pnp32-static-docs.pantheonsite.io/docs/performance/ so I'm not 100% sure performance & logs are right choice here.

Reply:
Cool! Is there a list of existing tags?

Reply:
@ari-gold I just shared with you a spreadsheet of all docs groups and tags.